src/components/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,13 @@
+import clsx from 'clsx';
+import type { FC } from 'react';
+import { memo } from 'react';
+import type { PropsWithChildrenAndClass } from 'src/utils/PropsWithClass';
+
+/** Skeleton component. */
```

Comment:
Please add todo to "decompose this component into text and rectangle components after designs are ready".

Reply:
I don't think we should decompose this component. It's better to add props for `text`/`rectangle` as other ui-kits do

Reply:
Good, then leave a todo comment about adding props after designs are ready